frontend/components/TableContainer/DataTable/LinkCell/LinkCell.tsx

```diff
@@ -4,9 +4,10 @@ import React from "react";
 import { Link } from "react-router";
 import classnames from "classnames";
 import TooltipWrapper from "components/TooltipWrapper";
+import TextCell from "../TextCell";
 
 interface ILinkCellProps {
-  value: string | JSX.Element;
+  value?: string | JSX.Element;
```

Comment:
render textcell instead of linkcell in the table config. I don't think it makes sense for link cell to handle an empty value case.